content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -711,7 +711,7 @@ UP BROADCAST RUNNING PROMISC MULTICAST  MTU:1460  Metric:1
 If you get this far, something very strange is happening.  Your Service is
 running, has Endpoints, and your Pods are actually serving.  You have DNS
 working, and `kube-proxy` does not seem to be misbehaving.  And yet your
-Service is not working.  You should probably let us know, so we can help
+Service is not working.  You should probably let us know, so you can help
```

Comment:
@thockin , I have not read through the entire page (yet), but just noticed this last s/we/you substitution 😃 .

Line 714: Question: Who should be contacted (a specific SIG?)
Possible edit: Let us know what is going on so that we can help investigate!

Also, line 136 has a spelling mistake, `Assuming everythign `

Reply:
Thanks.  Will make all these changes.  The contact points are just below this.